src/main/kotlin/blackjack/step2/domain/RandomCardPicker.kt

```diff
@@ -0,0 +1,23 @@
+package blackjack.step2.domain
+
+class RandomCardPicker : CardPicker {
+    override fun pick(): Card {
+        val cardNumber = CardNumber.entries.shuffled().first()
+        val cardType = CardType.entries.shuffled().first()
+
+        return Card(cardNumber, cardType)
+    }
+
+    override fun pick(count: Int): List<Card> {
+        require(count in MIN_CARD_COUNT..MAX_CARD_COUNT) { "카드는 1장 이상 52장 이하만 가능합니다." }
```

Comment:
```suggestion
        require(count in MIN_CARD_COUNT..MAX_CARD_COUNT) { "카드는 $MIN_CARD_COUNT장 이상 $MAX_CARD_COUNT장 이하만 가능합니다." }
```
구현하신 상수를 활용해보면 어떨까요?